doc/users/next_whats_new/fig_add_artist_method.rst

```diff
@@ -0,0 +1,16 @@
+Figure has an `~.figure.Figure.add_artist` method
```

Comment:
"an" -> "a"

Reply:
The `~` cuts it to just `add_artist`, so 'an' is correct here.